docs/topics/cache.txt

```diff
@@ -1178,45 +1178,34 @@ decorator)::
 
         return response
 
-There are a few other ways to control cache parameters. For example, HTTP
-allows applications to do the following:
-
-* Define the maximum time a page should be cached.
-
-* Specify whether a cache should always check for newer versions, only
-  delivering the cached content when there are no changes. (Some caches
-  might deliver cached content even if the server page changed, simply
-  because the cache copy isn't yet expired.)
-
-In Django, use the ``cache_control`` view decorator to specify these cache
-parameters. In this example, ``cache_control`` tells caches to revalidate the
-cache on every access and to store cached versions for, at most, 3,600 seconds::
+Generally, HTTP's caching mechanisms give you a lot of control over downstream
```

Comment:
Not sure if "Generally" adds anything. Maybe you could expand the phrase a bit or simply remove it.